packs/aws/sensors/sqs_sensor.py

```diff
@@ -32,9 +34,17 @@ def __init__(self, sensor_service, config=None, poll_interval=5):
                                            poll_interval=poll_interval)
 
     def setup(self):
-        self.input_queue = self._get_config_entry(key='keys ', prefix='sqs_sensor')
+        queues = self._get_config_entry(key='input_queues', prefix='sqs_sensor')
+
+        # XXX: This is a hack as from datastore we can only receive a string while
+        # from config.yaml we can receive a list
+        if type(queues) is str:
+            self.input_queues = queues.split(', ')
+        else:
```

Comment:
If it's not a string, I suggest raising a ValueError that it's not in the expected format.